src/schema/nmdc.yaml

```diff
@@ -727,6 +754,10 @@ slots:
     range: FluidHandling
     description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
 
+  has_configuration:
+    range: Configuration
+    description: a set of parameters that define the actions of the process
```

Comment:
I am going to merge this but make a PR afterwards, with you as an approver, to change this description. As a matter of principle, we can't use near-identical descriptions for `has_configuration` and and `Configuration`

Reply:
Sounds good, thanks.  I could definitely use support in proper/ontologically correct descriptions.